### PR TITLE
Feature/1248 new table option for terms display

### PR DESCRIPTION
**Pull request by Deji98**
## Description
I added new table option for terms display

## Applicable issues
#1248 

## Checklist

- [x] I have created a specific branch for this pull request before committing, starting off the current HEAD of `development` branch. 
- [x] I'm submitting to the `development`, feature/hotfix/release branch. (Do not submit to the master branch!)
- [x] This pull request relates to a specific problem (bug or improvement).
- [x] I have mentioned the issue number in the pull request description text.
- [x] All the issues mentioned in this pull request relate to the problem I'm trying to solve.
- [x] The code I'm sending follows the [PSR-12](https://www.php-fig.org/psr/psr-12/) coding style.

### CODE REVIEW

File: inc/autolinks.php [hunk 1]
```diff
@@ -983,7 +983,7 @@ public function taxopress_manage_autolinks()
                                                 </h3>
                                             </div>
                                             <div class="inside">
-                                                <p><?php echo sprintf(esc_html__('If your website language does not use the A-Z alphabet, %1s please read this documentation. %2s', 'simple-tags'), '<a href="https://taxopress.com/docs/characters/">', '</a>'); ?>
```
Comment (ojopaul) [inline inc/autolinks.php:986]: Did you work on this line? If not, why is the line reformatted? Please correct and remove your format

File: inc/autoterms.php [hunk 2]
```diff
@@ -1214,7 +1214,7 @@ class="page-title-action"><?php esc_html_e('Logs', 'simple-tags'); ?> <span clas
                                     </h3>
                                 </div>
                                 <div class="inside">
-                                    <p><?php echo sprintf(esc_html__('If your website language does not use the A-Z alphabet, %1s please read this documentation. %2s', 'simple-tags'), '<a href="https://taxopress.com/docs/characters/">', '</a>'); ?>
```
Comment (ojopaul) [inline inc/autoterms.php:1217]: Same as above, remove he reformatting if you didn't work on this line

File: inc/class.client.php [hunk 3]
```diff
@@ -63,6 +66,16 @@ public function __construct() {
 		return true;
 	}
 
+	function enqueue_frontend_scripts() {
+
+		wp_register_script('frontend.js', STAGS_URL . '/assets/js/frontend.js', array('jquery'), STAGS_VERSION);
```
Comment (ojopaul) [inline inc/class.client.php:71]: Just as I told you yesterday, do not use generic name as they'll conflict with other plugins. frontend.js and frontend.css are too generic as name to enqueue css and js.

Please always check existing code to see best practice especially when we've done similar thing before

**Review by ojopaul — CHANGES_REQUESTED** [review]
@Deji98 Please check my comments on the PR

File: inc/class.client.php [hunk 4]
```diff
@@ -63,6 +66,16 @@ public function __construct() {
 		return true;
 	}
 
+	function enqueue_displayformat_scripts() {
+
+		wp_register_script('taxopress-frontend.js', STAGS_URL . '/assets/frontend/js/frontend.js', array('jquery'), STAGS_VERSION);
```
Comment (ojopaul) [inline inc/class.client.php:71]: Remove the dot (.) from the name. Only use either hyphen or underscore to separate name just like we do for others

File: inc/class.client.php [hunk 5]
```diff
@@ -63,6 +66,16 @@ public function __construct() {
...
+		wp_register_style('taxopress-frontend.css', STAGS_URL . '/assets/frontend/css/frontend.css', array(), STAGS_VERSION, 'all');
```
Comment (ojopaul) [inline inc/class.client.php:72]: Same as above, the register name should be taxopress-frontend-css and not taxopress-frontend.css

**Review by ojopaul — CHANGES_REQUESTED** [review]
@Deji98 Attend to the new changes

File: inc/class.client.php (hunk 5 above)
Comment (Deji98) [reply inc/class.client.php]: Okay, done that.
Thank you

**Pull request closed without merging**